wicket-kendo-ui-culture/pom.xml

```diff
@@ -25,6 +25,12 @@
 			<artifactId>wicket-jquery-ui-core</artifactId>
 			<version>6.20.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>com.googlecode.wicket-jquery-ui</groupId>
+			<artifactId>wicket-kendo-ui</artifactId>
+			<version>6.20.1-SNAPSHOT</version>
+			<type>bundle</type>
```

Comment:
Is there any particular reason of specifying the type? I do not remember having used this in other places....